### PR TITLE
doc: add export `PERL5LIB` env for running test cases

### PR DESCRIPTION
**Pull request by totemofwolf**
### Full changelog

* add export `PERL5LIB` env for running test cases

### CODE REVIEW

File: doc/how-to-build-cn.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ Makefile rules:
 2. 然后通过 cpanm 来安装 test-gninx：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. 然后 clone 最新的源码：`git clone https://github.com/openresty/test-nginx.git`
 4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集：
+    * 追加当前目录到PERL模块目录：`export PERL5LIB=.:$PERL5LIB`
```
Comment (membphis) [inline doc/how-to-build-cn.md]: add a space between Chinese and English.

Comment (membphis) [reply doc/how-to-build-cn.md]: it seems `perl`  is better.

Comment (totemofwolf) [reply doc/how-to-build-cn.md]: ok

**Comment by moonming** [comment]
why we need this PR?

**Comment by sshniro** [comment]
This addition will be useful because for Ubuntu this env config is needed to run the test cases.

**Comment by totemofwolf** [comment]
> This addition will be useful because for Ubuntu this env config is needed to run the test cases.

So does MacOSX.

**Pull request merged**